addons/nametags/functions/fnc_onDraw3d.sqf

```diff
@@ -132,7 +132,15 @@ if (_enabledTagsNearby) then {
         if !(isNull _target) then {
             private _relPos = (visiblePositionASL _target) vectorDiff _camPosASL;
             private _distance = vectorMagnitude _relPos;
-            private _projDist = _relPos vectorDistance (_vecy vectorMultiply (_relPos vectorDotProduct _vecy));
+
+            private _screenPos = worldToScreen (_target modelToWorld (_target selectionPosition "head"));
+            if (_screenPos isEqualTo []) then {
```

Comment:
why continue the render process? this is wased time. if the Nametag is not on screen.